core/templates/pages/exploration-editor-page/feedback-tab/services/thread-data.service.ts

```diff
@@ -16,26 +16,31 @@
  * @fileoverview Service for getting thread data from the backend for the
  * feedback tab of the exploration editor.
  */
-
-require('domain/feedback_message/ThreadMessageObjectFactory.ts');
-require('domain/feedback_thread/FeedbackThreadObjectFactory.ts');
-require('domain/suggestion/SuggestionThreadObjectFactory.ts');
-require('domain/utilities/url-interpolation.service.ts');
-require(
-  'pages/exploration-editor-page/exploration-editor-page.constants.ajs.ts');
-require('pages/exploration-editor-page/exploration-editor-page.constants.ts');
-require('services/alerts.service.ts');
-require('services/context.service.ts');
-require('services/suggestions.service.ts');
-
-import { FeedbackThread, FeedbackThreadBackendDict } from
-  'domain/feedback_thread/FeedbackThreadObjectFactory';
-import { SuggestionThread } from
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+// Import cloneDeep from 'lodash/cloneDeep';
```

Comment:
Please don't add commented out code.

Reply:
Okay, will do that

Reply:
Also, I was assigned thread-data.service.ts only. I hope to be done with this and ask to be assigned to another one probably the spec file.